app/javascript/ui/casebooks/dragging.coffee

```diff
@@ -0,0 +1,97 @@
+ui = require 'ui/turbolinks'
```

Comment:
Cool, let's go with that. 

sprockets-es6 is just babel for sprockets, yeah? Seems like it doesn't provide module support, though... Are you thinking we'd just vendor dependencies and use sprockets requires? 

Alternative is we could keep using `yarn` with something simpler like `sprockets-commoner` to replace webpack.